coopersystem-fsd/pcc
# Added login button in navbar

## Description

- Added login/logout button in navbar
- Reduced spacing between menu items
- Added functions for login/logout via AJAX
- Fixed the width of the block in the view where it exceeded the width of the screen
- Added rules for displaying paid event for external users

## Steps to test

1. Log in and log out using the navbar modal

**Expected behavior:** If any information is entered incorrectly, a message will be displayed. If all goes well, the page will reload.

## Additional information

It is necessary to run the command `npm run build` to apply the styles correctly
